modules/ddl-and-loading/pages/functions/token/gsql_ts_to_epoch_legacy.adoc

```diff
@@ -1 +1,49 @@
 = gsql_ts_to_epoch_seconds_legacy()
+
+Converts a timestamp in canonical string format to Unix epoch time (seconds since `Jan 1, 1970`).
+If the timestamp is invalid or earlier than `1970-01-01 00:00:00`, the function returns a default value (0 by default), ensuring backward compatibility and preventing exceptions.
```

Comment:
```suggestion
If the timestamp is invalid or earlier than `1970-01-01 00:00:00`, the function returns a user-specified value (0 by default), ensuring backward compatibility and preventing exceptions.
```
The use of "default" two times can be confused (default if input is not value; default for the "default").